### PR TITLE
Vertically center Hedvig logo

### PR DESCRIPTION
**Pull request by robinandeer**
## What

Remove white space from bottom of svg.

## Why

The logo has some white space at the bottom so it doesn't look vertically centered.

https://user-images.githubusercontent.com/1220232/118952941-a87fa700-b95c-11eb-8f32-591fa7898936.mov

### CODE REVIEW

File: src/Components/Header.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ const LogoImage = styled.img`
 
 const Svg = styled.svg`
   fill: currentColor;
+  vertical-align: bottom;
```
Comment (gustaveen) [inline src/Components/Header.tsx:37]: Works perfect! Setting `display: flex` on the parent `<a>` could be an alternative

**Review by gustaveen — APPROVED** [review]
So nice that you can see stuff like that, I'm blind to that by now haha :D

File: src/Components/Header.tsx (hunk 1 above)
Comment (robinandeer) [reply src/Components/Header.tsx:37]: Nice, I took inspiration from here: https://stackoverflow.com/questions/7774814/remove-white-space-below-image

I thought this was the least intrusive way

Comment (gustaveen) [reply src/Components/Header.tsx:37]: Ah nice, so it actually aligns it along the baseline, I think it's just the semantics of `bottom` that confuses me :D

**Pull request merged**